heft-plugins/heft-sass-plugin/src/schemas/heft-sass-plugin.schema.json

```diff
@@ -32,6 +32,11 @@
       "description": "Determines if export values are wrapped in a default property, or not."
     },
 
+    "cssOutputFolder": {
+      "type": "string",
+      "description": "If specified, will output .css and .scss.json files."
```

Comment:
Are we outputting .scss.json files? In the heft-sass-test project, I'm seeing `.scss.css` and `.sass.css` files that correspond to the .scss and .sass style files in that project.

Reply:
Description is outdated. Will update.